tests/unit/test_lowpan.cpp

```diff
@@ -1832,11 +1832,83 @@ void TestLowpanIphc(void)
     testFreeInstance(sInstance);
 }
 
+void TestLowpanMeshHeader(void)
```

Comment:
Can we add some tests to ensure `ParseFrom` does not read beyond `headerLength`?

Reply:
Good idea. Will add it.